### ISSUE TITLE
gnome-wacom-properties

### ISSUE THREAD

**Issue by luke-titley-dwarf** [mate-desktop/mate-control-center#353]
Moving over to centos7 + MATE from centos6. We're an animation studio that has artists using wacom tablets and cintiq.
- Are there any efforts underway to provide a user interface for configuring wacom tablets/cintiqs in MATE. ie calibration/custom buttons.
- If such a thing were to be written (quickly) are there any particular large issues you guys can foresee? I was thinking of writing a quick UI perhaps in python perhaps in c++ that would do the basics of screen to tablet calibration, rotation and custom buttons.

Is there any point? Is someone already working on this?

Would appreciate any feedback you guys have about this.

**Comment by sebastianelsner** [comment]
Hello, I am in the same situation. @luke-titley-dwarf  what was your final solution?

**Comment by raveit65** [comment]
> Is there any point? Is someone already working on this?

no, we are a small team....

**Comment by vkareh** [comment]
If anyone wants to work on a backport, help would be appreciated. Here's the relevant code:

* [gnome-control-center](https://gitlab.gnome.org/GNOME/gnome-control-center/tree/master/panels/wacom)
* [gnome-settings-daemon](https://gitlab.gnome.org/GNOME/gnome-settings-daemon/tree/master/plugins/wacom)

**Comment by luke-titley-dwarf** [comment]
@sebastianelsner, final solution was to use this application. With some modifications to allow us to calibrate. After a bit of testing we plan to 'pull request' our changes.
https://github.com/tb2097/wacom-gui.git

**Comment by mjnowen** [comment]
Did this pull request ever happen?

**Comment by luke-titley-dwarf** [comment]
No, it didn't but I can kick the changes over to you, if you want them.

**Comment by mjnowen** [comment]
Yeah, that would be cool. Thanks. Any reason to not just share via a pull request for everyone? Appreciate your efforts here, sir!

**Comment by schizogeny** [comment]
Hey, just posting to add a vote for this. MATE is the most even-tempered DE around but the wacom util is solely on GNOME and KDE. GNOME is so condescending while KDE was several times buggier than MATE I couldn't believe it (it's 2020), now I have to use a Windows machine.

If someone from the MATE team accepted to take this on and get it to work well, I would buy you a small Wacom Intuos Pro tablet (same as mine, retails 250$ USD on amazon). My experience dev'ing linux apps specifically is too low, I don't think I'll have time to learn.

Bonus point: If someone can figure out (couldn't in GNOME) how to configure the pen tracking max lift height between the pen tip and tablet (the height after which it stops tracking and moving the cursor - the max distance between pen tip and tablet, I don't know the term, I'm trying - I am _not_ experienced with these tablets), we can sweeten the pot, maybe a smaller tablet model (let's say a small regular Intuos or other model 100$ USD or less) so you have more than one model to test, or for someone else or whoever figures it out, or something or your choice. 

No specific time limit on this offer but it would be nice to have in an upcoming Debian stable or Ubuntu LTS release.

**Comment by sbuttie** [comment]
I've just bought a basic Wacom tablet and hit this problem too.  A work around for the technically inept like me is to boot using Cinnamon, which has the hardware option to configure a graphics tablet, set up your tablet, then reboot into Mate.  The tablet works, though obviously you can't tweak settings on the fly.  Good luck with a proper fix for Mate, and thanks.

**Comment by bastianlstrube** [comment]
Same problem here. Tablet working on MATE machine but no way to configure it. I'm a lefty too so really need to change things... looking at [wacom-gui](https://github.com/tb2097/wacom-gui) it needs python-qt4 from old repositories. 
Would love to help but have only a experience with python as a scripting tool for 3D software.

**Comment by blendernoob** [comment]
I second adding a tablet gui to MATE. With so many DCCs still being X11 only, and how easy it is to customize the desktop, MATE is one of the best and most stable X11 DEs for artists. If this is not possible or there is not enough interest, I wonder how people have worked around this? I saw that [some folks at Disney managed to get the KDE settings menu to work in MATE](https://www.reddit.com/r/linux/comments/1mokkt0/comment/n8kmtyr/?context=3), and I wonder if this is possible with Plasma 6's settings menu and MATE 1.28. I was considering open tablet driver but that conflicts with GNOME and KDE's tablet driver. Also the [wacom-gui](https://github.com/tb2097/wacom-gui) relies on very old dependencies that are not available in Fedora anymore.

**Comment by lukefromdc** [comment]
Problem is we are a very small team right now. In my case a further complication is I have no online finanical accounts and thus can neither send nor receive money, so no way anyone can buy me gear unless we are in the same area.
